zulipterminal/ui_tools/views.py

```diff
@@ -1602,7 +1602,7 @@ def __init__(
 
         # To keep track of buttons (e.g., button links) and to facilitate
         # computing their slice indexes
-        button_widgets = []  # type: List[Any]
+        self.button_widgets = []  # type: List[Any]
```

Comment:
Was `button_widget` made into an attribute only so that it can be accessed from tests? If so `MsgInfoView.log` or `MsgInfoView.body` would be better candidates for doing the test instead of `button_widget`